src/content/app/entity-viewer/gene-view/components/protein-domain-image/ProteinDomainImage.tsx

```diff
@@ -51,7 +56,7 @@ export const getDomainsByResourceGroups = (
 
   proteinDomains.forEach((domain) => {
     const {
-      sequence_family: { description: domainName },
+      sequence_family: { name: domainName, description: description },
```

Comment:
Just `description` should do here and below

Reply:
@ridwanamo This hasn't been changed yet. It isn't a very important one, but do you intend to keep it this way?

Reply:
@imransl sorry i missed that. I have now pushed this change.